Dev/config/strings.js

```diff
@@ -130,6 +130,9 @@ export default {
     Whoops: 'Whoops',
     AddStudents: 'Add Students',
     JoinMyClass: 'Join my class on QuranConnect. Here is the class code: ',
+    AddYourStudents: 'Add Your Students',
+    Or: 'Or',
+    AddStudentsManually: 'Manually Add Your Students',
```

Comment:
I'd only capitalize the first letter when it is a full sentence. Same thing for Add your students